department-of-veterans-affairs/caseflow
# Div/appeals 42524

Resolves [Response Letters: Can view response letter information](https://jira.devops.va.gov/browse/APPEALS-42524)

# Description
Please explain the changes you made here.

## Acceptance Criteria

## Testing Plan

1. Login to caseflow as INBOUND_OPS_TEAM_ADMIN_USER 
2. Navigate to Queue
3. Click the Switch Views dropdown and select Correspondence Cases.
4. Select any correspondence from Assigned tab and navigate to intake page.
5. From there, click **Add letter** button and fill out the form with the date sent less than 10 days from today. Select the radio field to custom and enter 2. Fill out the other necessary fields as required. This is to get the Expired on in the response letter details page. 
6. Also, Add letters to other 2 orders, like X day(s) left and No response window required options.
7. After adding, navigate to the details page for that correspondence and click the Response Letter tab to verify the data.
8. Please see the above screenshots how it looks like


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
 
<img width="1482" alt="Screenshot 2024-08-14 at 8 11 41 AM" src="https://github.com/user-attachments/assets/ed534889-9231-484d-aafd-2c369540535c">
<img width="1421" alt="Screenshot 2024-08-14 at 8 11 58 AM" src="https://github.com/user-attachments/assets/a9492888-afd9-4954-95a9-1b36fae131ac">


## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
